### ISSUE TITLE
Exporting a static model with batch size greater than 1

### ISSUE THREAD

**Issue by Mahaaveer** [ArgoHA/custom_d_fine#39]
Greetings !

Great work on implementing D-FINE from the scratch, This repo is much easier to use than the native d-fine repo.
As it has been already established, Exporting Transformers based models with dynamic axes deteriorates accuracy, hence i wanted to try and benchmark batched inference with a static model of batch size 4.  

I tried exporting the model with the following settings:

export:
  half: true
  max_batch_size: 4
  dynamic_input: false

export script produced a model.onnx with the following shapes:

name: input tensor: float32[s77,3,640,640] 

name: logits tensor: float32[s77,300,3] 
name: boxes tensor: float32[s77,300,4] 

I believe s77 has something to do with dynamic axes and this export breaks it.

Is it possible to export D-fine models with static batch size greater than 1  ?

Thanks!
